### PR TITLE
Dockerize tutorial

### PR DESCRIPTION
**Pull request by Bogdaan**
I think, this tutorial will be even better if we add docker files.

@yolossn what do you think?

### CODE REVIEW

**Comment by yolossn** [comment]
Hi, thanks for raising this PR. It is a good Idea to dockerize this setup. Can you also add `Grafana` to the docker-compose file?

**Comment by Bogdaan** [comment]
added

**Comment by yolossn** [comment]
I will test this once by EoD and merge this PR.

**Comment by yolossn** [comment]
The docker build failed with the following error.

```
serverWithMetric.go:7:2: cannot find package "github.com/prometheus/client_golang/prometheus" in any of:
        /usr/local/go/src/github.com/prometheus/client_golang/prometheus (from $GOROOT)
        /go/src/github.com/prometheus/client_golang/prometheus (from $GOPATH)
```

Please add a `go.mod` that should fix this.

File: README.md [hunk 1]
```diff
@@ -15,7 +15,8 @@
 4. [Show me how it is done](https://github.com/yolossn/Prometheus-Basics#show-me-how-it-is-done)
 5. [Types of metrics](https://github.com/yolossn/Prometheus-Basics#types-of-metrics)
 6. [Create a simple exporter](https://github.com/yolossn/Prometheus-Basics#create-a-simple-exporter)
-7. [References](https://github.com/yolossn/Prometheus-Basics#References)
+7. [Run prometheus in docker](https://github.com/yolossn/Prometheus-Basics#run-prometheus-in-docker)
```
Comment (yolossn) [inline README.md]:
```suggestion
7. [Run prometheus and grafana in docker](https://github.com/yolossn/Prometheus-Basics#run-prometheus-and-grafana-in-docker)
```

File: README.md [hunk 2]
```diff
@@ -432,15 +433,24 @@ I hope I did justice to your time and helped you understand the basics of promet
...
-
-
 To Do 
 - [x] Integration with grafana to create dashboards
 - [ ] Add code samples for all metric types.
 - [ ] Explain about the concept of Service Discovery for integrating with kubernetes.
 - [ ] Basic Alerting + Prometheus alerts vs Grafana alerts.
 - [ ] Integrating alerts with tool like pagerduty.
 
+### Run prometheus in docker
```
Comment (yolossn) [inline README.md]:
```suggestion
### Run prometheus and grafana in docker
```

File: README.md [hunk 3]
```diff
@@ -432,15 +433,24 @@ I hope I did justice to your time and helped you understand the basics of promet
...
+
+You can run example in docker:
```
Comment (yolossn) [inline README.md]: Please be a little more elaborative here what all services will be run using docker also I would suggest adding a link to the docker-compose file.

Comment (Bogdaan) [reply README.md]: Sure, propose you changes.

**Comment by Bogdaan** [comment]
> Please add a `go.mod` that should fix this.

added

File: README.md (hunk 3 above)
Comment (yolossn) [reply README.md]:
```suggestion
You can run the whole setup of simple ping server, Prometheus and Grafana using docker. Just run the following command.
```

**Comment by yolossn** [comment]
@Bogdaan Thanks for contributing.

**Pull request merged**